etcd/tnf/v1alpha1/tests/pacemakerstatuses.tnf.etcd.openshift.io/DualReplica.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "PacemakerStatus"
+crdName: pacemakerstatuses.tnf.etcd.openshift.io
+featureGates:
+- DualReplica
+tests:
+  onCreate:
+    - name: Should be able to create a minimal PacemakerStatus
+      initial: |
+        apiVersion: tnf.etcd.openshift.io/v1alpha1
+        kind: PacemakerStatus
+        metadata:
+          name: cluster
+        spec: {}
+      expected: |
+        apiVersion: tnf.etcd.openshift.io/v1alpha1
+        kind: PacemakerStatus
+        metadata:
+          name: cluster
+        spec: {}
```

Comment:
This may fail validation, we could do add a spec field to the type definition:

```go
  // spec is reserved for future use but is currently unused.
  // +optional
  Spec PacemakerStatusSpec `json:"spec,omitempty"`
```

And define the spec type (even if empty):
```go

  // PacemakerStatusSpec is reserved for future use
  type PacemakerStatusSpec struct {
      // Reserved for future use
  }
```

Reply:
Added. :) I was hoping to get away without one, but I'm fine with it just being empty.